core/include/detray/materials/material.hpp

```diff
@@ -19,6 +19,13 @@
 
 namespace detray {
 
+namespace detail {
+
+/// Tag a class as 'simple material'
+struct simple_material_tag {};
```

Comment:
I might need to continue reading to understand this - but what does this do exactly? And _simple_ is to indicate the thin scatter?

Reply:
In that case, I would call it sth like homogeneous_material or mono_material 
But we also need to think about how the material class with material_map would look like in the actual code base